recipe_scrapers/saltpepperskillet.py

```diff
@@ -8,3 +9,24 @@ def host(cls):
 
     def author(self):
         return self.schema.author().title()
+
+    def ingredient_groups(self):
+        return group_ingredients(
+            self.ingredients(),
+            self.soup,
+            ".wprm-recipe-ingredient-group h4",
+            ".wprm-recipe-ingredient",
+        )
```

Comment:
At some point we should probably consider making WPRM ingredient-group support available as a [`Mixin`](https://www.ianlewis.org/en/mixins-and-python) (or perhaps a few mixins with different variants based on the WordPress plugin(s) in use) -- these defaults seem to appear fairly often.